README.md

```diff
@@ -12,16 +12,16 @@ The documentation is available at [trento-project.io/wanda](https://trento-proje
 
 Swagger UI is available at [trento-project.io/wanda/swaggerui](https://trento-project.io/wanda/swaggerui).
 
-## Developing Checks
+# Developing Checks
 
 Wanda architecture aims to simplify [testing Checks Executions](#testing-executions) and [adding new ones](#adding-new-checks).
 
-### Infrastructure
+## Infrastructure
 
-For development purposes, a [docker-compose file](https://github.com/trento-project/wanda/blob/main/docker-compose.yaml) is provided.
+For checks development purposes, a [docker-compose file](https://github.com/trento-project/wanda/blob/main/docker-compose.yaml) is provided.
```

Comment:
I think it was good before without the "checks" word in here. While docker-compose.yaml file gives us a basic environment to work on wanda itself, the `docker-compose.checks.yaml` give us additionally wanda running in container too, ideal for the development of checks